Resources/doc/dynamic.md

```diff
@@ -163,6 +163,63 @@ php bin/adminconsole debug:container --tag=sulu_form.dynamic.type
 
 **Provider for type "structure" already exists**
 
+## Add Honeypot field for spam protection
+
+If you want to add a honeypot field for spam protection you can activate it the following way:
+
+```yaml
+sulu_form:
+    honeypot:
+        field: "Honey Pot Field Name"
+        strategy: spam # no_save, no_email, spam
+```
+
+There are 3 honeypot strategies:
+
+**spam**
+
+Spam is the default behaviour and it will just mark the email subject with a prefix `(SPAM) `.
+
+**no_email**
+
+This option will save the form if enabled but will not send any emails.
+
+**no_save**
+
+Will not save and will not send any emails.
+
+### Style honey pot field
+
+The honey pot field need to be hidden so in your theme add a new class to it:
+
+```twig
+{%- block form_row -%}
+    {% if form.vars.attr['honeypot']|default(false) %}
+        <div class="honung">
+    {% endif %}
+
+    {{ block('sulu_form_row_start') }}
+    {{- form_label(form) -}}
+    {{- form_errors(form) -}}
+    {{- form_widget(form) -}}
+    {{ block('sulu_form_row_end') }}
+
+    {% if form.vars.attr['honeypot']|default(false) %}
+        </div>
+    {% endif %}
+{%- endblock -%}
+
+```
+
+And then hide it in your css with:
+
+```css
+.honung {
+    display: none;
+}
+```
+
```

Comment:
This should be the same as here: https://github.com/sulu/SuluFormBundle/blob/master/Resources/doc/dynamic.md#style-honey-pot-field

The block name should be something like `_honeypot_row`